root/static/scripts/release-editor/trackParser.js

```diff
@@ -485,17 +485,19 @@ trackParser.customDelimiterError = debounce(function () {
 })
 
 function optionCookie(name, defaultValue, checkbox=true) {
-    var existingValue = getCookie(name);
-
-    if (checkbox) {
-      var observable = ko.observable(
-          defaultValue ? existingValue !== "false" : existingValue === "true"
-      );
-    } else {
-      var observable = ko.observable(
-          existingValue ? existingValue : defaultValue
-      );
-    }
+    const existingValue = getCookie(name);
+
+    const observable = checkbox  
+          ? ko.observable(
+              defaultValue  
+                  ? existingValue !== "false"
+                  : existingValue === "true"
+            )
+          : ko.observable(
+              existingValue 
+                  ? existingValue 
+                  : defaultValue
+            )
```

Comment:
Missing semi-colon here:
```suggestion
            );
```